experimental/DoubleAndHyperComplexes/test/free_resolutions.jl

```diff
@@ -19,3 +19,21 @@
   minimal_betti_table(FI)
   simplify(FI)
 end
+
+@testset "Betti tables over quotient rings" begin
+  R, (x, y, z) = graded_polynomial_ring(QQ, [:x, :y, :z])
+  I = ideal(R, z)
+  A, _ = quo(R, I)
+  (x, y, z) = gens(A)
+
+  F = graded_free_module(A, 1)
+
+  J, inc = sub(F, [x*F[1], y*F[1]])
+
+  M = cokernel(inc)
+
+  res, aug = free_resolution(Oscar.SimpleFreeResolution, M)
+
+  betti_table(res; upper_bound=3)
+  minimal_betti_table(res; upper_bound=3)
```

Comment:
maybe add a "real" test to catch regressions/changes instead of only testing that it doesn't throw any errors.

Reply:
I suppose the Berlin crew has some use cases from which we can cook up real tests. This was only to assure it doesn't throw errors and to give them an idea for how to use this stuff. We can change all this later in case people are really happy with this. If not: no need to put in all this extra work at this point.